core/local/steps/dispatch.js

```diff
@@ -117,6 +120,9 @@ actions = {
       return actions.createddirectory(event, {prep})
     }
     const doc = buildDir(event.path, event.stats)
+    if (event.overwrite) {
+      doc.overwrite = event.overwrite
```

Comment:
Why do we `fetchOldDoc()` for files but not for dirs?
Also isn't *Merge* expecting a boolean and overwriting with a doc anyway?